core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -153,10 +156,15 @@ func (env *CouchDBCommonStorageTestEnv) Init(t testing.TB) {
 		stateDBConfig,
 		[]string{"lscc", "_lifecycle"},
 	)
+	logger.Infof("processing")
+	if err != nil {
+		logger.Infof(err.Error())
+	}
 	assert.NoError(t, err)
 	env.t = t
 	env.provider = dbProvider
-	env.redoPath = redoPath
```

Comment:
i'm not sure if `redoPath` is actually being used... @manish-sethi can we safely delete it from struct?

Reply:
This is getting used in the cleanup function for deleting the redoPath dir